recipes/clsparse/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "clSPARSE" %}
+{% set version = "0.10.2.0" %}
+{% set sha256 = "952b26670625646c5386299a4d360aaa9cd2139555376b48aaed163a6ea60325" %}
+
+{% set OSX_VARIANT = os.environ.get("OSX_VARIANT", "native") %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/clMathLibraries/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script_env:
+    - OSX_VARIANT
+
+{% if OSX_VARIANT != "native" %}
+  skip: True  # [not osx]
+{% else %}
+  skip: True  # [win and py<36]
+{% endif %}
+
+  features:
+{% if OSX_VARIANT == "pocl" %}
+    - osx_pocl_opencl            # [osx]
+{% endif %}
+    - vc14                       # [win and py36]
+
+requirements:
+  build:
+    - toolchain
+    - python                     # [win]
+    - cmake
+    - boost-cpp
+    - clhpp
+    - gtest
+    - ocl-icd                    # [linux]
+    - khronos-opencl-icd-loader  # [win]
+{% if OSX_VARIANT == "pocl" %}
+    - osx-pocl-opencl            # [osx]
+    - pocl >=0.14                # [unix]
+{% endif %}
+
+  run:
+    - khronos-opencl-icd-loader  # [win]
+    - ocl-icd                    # [linux]
+{% if OSX_VARIANT == "pocl" %}
+    - osx-pocl-opencl            # [osx]
+    - pocl >=0.14                # [unix]
+{% endif %}
+
+test:
+  requires:
+    - python {{ environ['PY_VER'] + '*' }}              # [win]
+
+  commands:
+    # Check files
+    - test -f "${PREFIX}/include/clSPARSE.h"           # [unix]
+    - test -f "${PREFIX}/lib/libclSPARSE.so"           # [linux]
+    - test -f "${PREFIX}/lib/libclSPARSE.dylib"        # [osx]
+    - if not exist %LIBRARY_INC%\\clSPARSE.h exit 1    # [win]
+    - if not exist %LIBRARY_BIN%\\clSPARSE.dll exit 1  # [win]
+    - if not exist %LIBRARY_LIB%\\clSPARSE.lib exit 1  # [win]
+
+about:
+  home: https://github.com/clMathLibraries/{{ name }}
+  license: Apache 2.0
+  license_file: LICENSE
+  summary: A software library containing Sparse functions written in OpenCL
+
+extra:
+  recipe-maintainers:
+    - inducer
```

Comment:
Have added you as a maintainer here per your [request]( https://github.com/conda-forge/staged-recipes/issues/3186#issuecomment-312209740 ), @inducer.